core/modules.js

```diff
@@ -30,7 +30,7 @@ exports.loadCoreModule = function(platform, module) {
         index = Object.keys(require.cache).indexOf(fp),
         m = index !== -1 ? require.reload(fp) : require.once(fp);
     m.platform = exports;
-    m.name = module;
+    m.name = module.split(".")[0];
```

Comment:
To remove the .js on the end of the name, other wise !help creator is not recognised, instead !help creator.js would be required, can store extra variable if you prefer, just made the name inline with a normal module